openshift/04-deployment-template.yml

```diff
@@ -103,6 +110,13 @@ objects:
           - containerPort: 8090
             name: management
             protocol: TCP
+          resources:
```

Comment:
@mikz Should we set the `APICAST_WORKERS` too?

Reply:
Nope. APIcast autodetects number of requested CPU cores: https://github.com/3scale/apicast/pull/600

Reply:
ok